lib/csu/aarch64c/crt1_c.c

```diff
@@ -68,12 +68,11 @@ Elf_Auxinfo *__auxargs;
  * This restriction only applies to statically linked binaries since the dynamic
  * linker takes care of initialization otherwise.
  */
-void
-_start(void *auxv,
+__dead2 static void
```

Comment:
I think you don't need the explicit `__dead2` as `__libc_start1` is already tagged with that so the compiler will infer it automatically.  That said, if you do keep it, I would spell this as  either`static void __dead2` or `static __dead2 void`.  Those are the predominate styles in the tree (a few more of the former).